### DISCUSSION TITLE
求助，如何显示通讯作者？

### DISCUSSION THREAD

**Discussion by MarkccDD** [MuiseDestiny/zotero-style#1003] [Q&A]
在创建者那一栏，输入什么语句可以显示通讯作者，目前显示的是第一作者（${firstCreator}），谢谢您！

**Comment by X-zhaotun** [comment]
下面的列设置，slices设置为-1

**Reply by MarkccDD** [reply]
谢谢 zhaotun，谢谢！但是设置之后还是仅显示第一作者（和 等），部分仅有两个作者的条目会显示一作和二作
